src/frontend/src/app/pages/open-proposal-details/open-proposal-details.component.ts

```diff
@@ -0,0 +1,214 @@
+        border-top: 1px solid $slate-300;
+
+        @include dark {
+          border-color: $slate-700;
+        }
+      }
+
+      .proposal-summary__content {
+        background-color: $slate-200;
+
+        @include dark {
+          background-color: $slate-900;
+        }
+      }
+    `,
+  ],
+  template: `
+    @if (currentProposal$ | async; as proposal) {
+      <h2 class="h3">{{ proposal.title }}</h2>
+
+      <app-card class="proposal">
+        <app-key-value-grid [columnNumber]="2">
+          <app-key-col id="proposal-id">ID</app-key-col>
+          <app-value-col aria-labelledby="proposal-id">
+            <a
+              href="{{ linkBaseUrl.ProposalId }}{{ proposal.id }}"
+              target="_blank"
+              rel="nofollow noreferrer"
+              >{{ proposal.id }}
+            </a>
+          </app-value-col>
+
+          <app-key-col id="proposal-links">Voting links</app-key-col>
+          <app-value-col
+            class="proposal-links"
+            aria-labelledby="proposal-links"
+          >
+            @if (proposal.proposalLinks !== []) {
+              @for (
+                proposalLink of proposal.proposalLinks;
+                track proposalLink.type
+              ) {
+                <a
+                  class="proposal-links__link"
+                  href="{{ proposalLink.link }}"
+                  target="_blank"
+                  rel="nofollow noreferrer"
+                  >{{ proposalLink.type }}
+                </a>
+              }
+            }
+          </app-value-col>
+
+          <app-key-col id="proposal-topic">Topic</app-key-col>
+          <app-value-col aria-labelledby="proposal-topic">
+            {{ proposal.topic }}
+          </app-value-col>
+
+          <app-key-col id="proposal-type">Type</app-key-col>
+          <app-value-col aria-labelledby="proposal-type">
+            {{ proposal.type }}
+          </app-value-col>
+
+          <app-key-col id="proposal-created">Created</app-key-col>
+          <app-value-col aria-labelledby="proposal-created">
+            {{ proposal.proposedAt | date: 'medium' }}
+          </app-value-col>
+
+          <app-key-col id="proposal-proposer">Proposer</app-key-col>
+          <app-value-col aria-labelledby="proposal-proposer">
+            <a
+              href="{{ linkBaseUrl.Neuron }}{{ proposal.proposedBy }}"
+              target="_blank"
+              rel="nofollow noreferrer"
+              >{{ proposal.proposedBy }}
+            </a>
+          </app-value-col>
+
+          <app-key-col id="proposal-review-end">Review period end</app-key-col>
+          <app-value-col aria-labelledby="proposal-review-end">
+            {{ proposal.reviewPeriodEnd | date: 'medium' }}
+          </app-value-col>
+
+          <app-key-col id="proposal-voting-end">Voting period end</app-key-col>
+          <app-value-col aria-labelledby="proposal-voting-end">
+            {{ proposal.votingPeriodEnd | date: 'medium' }}
```

Comment:
If we use this same format in multiple places, it may be worth creating our own pipe that wraps the date pipe and passes in the common format.